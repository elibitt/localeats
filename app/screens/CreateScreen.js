import React from 'react';
import {
  Image,
  Platform,
  ScrollView,
  StyleSheet,
  Text,
  TouchableOpacity,
  View,
} from 'react-native';
import { ExpoLinksView } from '@expo/samples';
import { ImagePicker } from 'expo';

pickImage = async () => {
  const result = await ImagePicker.launchImageLibraryAsync({
    allowsEditing: true,
    base64: true,
  });
  if (!result.cancelled) {
    this.setState({
      image: result.uri,
    });
  }
};

export default class LinksScreen extends React.Component {
  static navigationOptions = {
    title: 'Create New Meal',
  };

  render() {
    return (
      <ScrollView style={styles.container}>
        <Card>
          <Input label="cuisine"  placeholder="Cuisine" />
          <Input label="seats"    placeholder="Open Seats " />
          <TextInput
            label="description"
            placeholder="Describe your meal..."
            multiline={true}
            numberOfLines={4}
          />
          <Button
            buttonStyle={{ marginTop: 20, backgroundColor: "#03A9F4" }}
            title="SIGN IN"
            onPress={() => {
              onSignIn().then(() => navigation.navigate("SignedIn"));
            }}
          />
        </Card>
      </ScrollView>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    paddingTop: 15,
    paddingLeft: 15,
    backgroundColor: '#fff',
  },
});
