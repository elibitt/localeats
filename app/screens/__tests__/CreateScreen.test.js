import React from 'react';
import renderer from 'react-test-renderer';

import CreateScreen from '../CreateScreen';

describe('Create meal screen', () => {
  it('correctly renders', () => {
    const tree = renderer.create(<CreateScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
