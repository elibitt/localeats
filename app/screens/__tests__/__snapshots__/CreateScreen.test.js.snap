// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create meal screen correctly renders 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "backgroundColor": "#eee",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "bottom": 0,
      }
    }
  />
</View>
`;
